go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/NYTimes/gziphandler v1.1.1 // indirect
 	github.com/agiledragon/gomonkey/v2 v2.2.0
 	github.com/aws/aws-sdk-go v1.30.9
+	github.com/blang/semver/v4 v4.0.0
```

Comment:
Can we use the already existing `github.com/Masterminds/semver` instead?

Reply:
yes, i have optimized.